snap/wrappers/juju

```diff
@@ -7,11 +7,11 @@ snap_lxd_socket=/var/snap/lxd/common/lxd
 
 lxd_binary=$(which lxd)
 
-# Prefer debian installed lxd, if availible
-if [ "$lxd_binary" = "/usr/bin/lxd" ]; then
-    export LXD_DIR=$debian_socket
-elif [ "$lxd_binary" = "/snap/bin/lxd" ]; then
+# Prefer snap installed lxd, if availible
+if [ "$lxd_binary" = "/snap/bin/lxd" ]; then
     export LXD_DIR=$snap_lxd_socket
+elif [ "$lxd_binary" = "/usr/bin/lxd" ]; then
+    export LXD_DIR=$debian_socket
```

Comment:
I thought we were no longer ever using non-snap lxd.  Should we just drop this case altogether?

Reply:
my only concern with this one is someone who may use juju outside of conjure-up?